### DISCUSSION TITLE
DIInstall missing

### DISCUSSION THREAD

**Discussion by torpak** [LunarVim/LunarVim#2325] [Q&A] [answered]
I just installed LunarVim stable. I tried configuring it for rust, but i got stuck at DIInstall.
`E492: Not an editor command: DIInstall codelldb`
Why is it missing?

**Comment by KevinSilvester** [comment, marked as answer]
The plugin DAPInstall is disabled by default.
You can enable it by adding `lvim.builtin.dap.active = true` in you config.lua

**Reply by joewarrenmeeks** [reply]
Hmm, I've added that line to my `.config/lvim/config.lua` and I've run `:PackerInstall` but I still get the above error when I'm trying to run `:DIInstall python`. 

This is a fresh install of LunarVim. Am I missing a step?

Thanks,

**Reply by adriancmiranda** [reply]
I think you should run `:PackerUpdate` instead

**Comment by joewarrenmeeks** [comment]
Nice thanks!
